tests/commontests.py

```diff
@@ -262,3 +262,9 @@ def _switch_revision_and_subdir(self, use_cache=True):
             ],
             use_cache
         )
+
+    def test_sslverify_disabled(self):
+        self.tar_scm_std('--sslverify', 'disable')
```

Comment:
Thanks a lot for adding these tests, however currently they only test that tar_scm doesn't barf when passed this option. It should also test that when the option is used, the backend SCM command is invoked with the appropriate option.  Please could you extend it to do that?  I think it should be quite easy since you can reuse existing techniques from `testassertions.py`, e.g. see how `assertRanInitialClone` tests that the initial `git clone` / `bzr checkout` etc. get run.  Thanks!